Cargo.toml

```diff
@@ -5,9 +5,16 @@ authors = ["Laurence Tratt <laurie@tratt.net>"]
 license = "Apache-2.0/MIT"
 edition = "2018"
 
+[[test]]
+name = "lang_tests"
+path = "lang_tests/run.rs"
+harness = false
+
 [dependencies]
 getopts = "0.2"
 libc = "0.2"
 
 [dev-dependencies]
-tempfile = "3.0"
+lazy_static = "1.3"
```

Comment:
You could (optionally) run `cargo update`. For example lazy_static 1.4 is available (although I think it's semantically compatible with 1.3 anyway?)

Reply:
`cargo update` won't do anything here, though `cargo outdated` would. Fixed in https://github.com/softdevteam/ykbf/pull/4/commits/dba29da3bde15aba3b42054e36e59ef1dd8043e1.

Reply:
Your are indeed correct.